contents/blog/transparent-enterprise-pricing.md

```diff
@@ -0,0 +1,125 @@
+
+That's why we made our core paid plans totally self-serve and free up to 1 million events per month, with the "per event" price decreasing as usage increases, as you can see below.
+
+![PostHog pricing calc](../images/blog/enterprise-pricing-blog/pricing-calc.png)
+
+This model has worked incredibly well for us. 
+
+It's taken us from nothing to significant revenue, and healthy growth, in less than two years without any outbound sales. 
+
+![PostHog's revenue going sharply up and to the right](../images/blog/pricing-lesson/revenue-pricing.png)
+
+Our customer success team works hard to help new and potential customers get what they need from PostHog, but we don't cold email companies or spend time chasing specific customers.
+
+## Adopting transparent enterprise pricing
+
+The last frontier for us was our Enterprise plans. While some companies practice partial transparency, we wanted 100% transparency across the board. This took a while to achieve, but we're glad we did.
+
+Our plans are split by whether you want to self-host PostHog, or have it hosted and managed by us.
+
+There are three options for [self-hosting PostHog](/pricing):
+
+- **Open-Source:** Contains all of the base analytics features but is limited to one project
+- **Scale:** Contains advanced features such as experiments and group analytics, and has unlimited projects
+- **Enterprise:** Comes with SAML-based SSO, project permissions and a whole lot of help from us
+
+For PostHog Cloud, there are just two options:
+
+- **Cloud:** Mirrors the features of PostHog Scale, hosted by us
+- **Cloud Enterprise:** Mirrors the features of PostHog Enterprise, hosted by us
+
+PostHog Scale and Cloud have had transparent pricing for a while and, unsurprisingly, we found companies interested in PostHog Enterprise would calculate the price using the Scale or Cloud pricing calculators. 
+
+This makes sense in isolation, but our enterprise plans cost more because you get more, particularly on self-hosted where we offer a lot of support to ensure things run smoothly.
+
+We adapted our existing pricing models to be roughly 2x Scale pricing for self-hosted Enterprise, and 1.3x Cloud pricing for Cloud Enterprise to reflect the additional benefits you gain from these editions.
+
+We didn't want to mandate lengthy annual commitments, so all of our products can be subscribed to on a monthly basis and canceled at any time.
+
+That said, most larger organizations fall into a traditional annual buying process, and we incentivize them with bigger discounts for the upfront commitment.
+
+## What did we learn?
+
+People expect self-hosted deployments to be cheaper than PostHog Cloud as they incur the additional infrastructure 
+costs. 
+
+While it's true that running PostHog Cloud requires a minimum level of spend from us, self-hosting costs more than Cloud for a few important reasons:
+
+1. Self-hosting bestows unique benefits you can't get with any other equivalent product analytics platform
```

Comment:
This basically says 'we charge more because you're stuck with us'. I think points 2 and 3 are more user-friendly, so I'd maybe put this 3rd or remove it? (Also 'bestows' - not plain English)